src/terms.jl

```diff
@@ -3,9 +3,15 @@
 """
     struct Term
 
-Represent a term symbol ``{}^{2S+1}L_{J}`` with specific parity in LS-coupling. As determining
-valid ``J`` values is simple for given ``S`` and ``L`` (``|L - S| \\leq J \\leq L+S``), it
-is not specified.
+Represents a term symbol with specific parity in LS-coupling.
+
+Only the ``L``, ``S`` and parity values of the symbol ``{}^{2S+1}L_{J}`` are specified. To
+specify a _level_, the ``J`` value would have to be specified separately. The valid ``J``
```

Comment:
Maybe then add a note that when used in a CSF, the level can be represented by `Level`?

Reply:
Which of course is undocumented...

Reply:
I might leave it out for now. Also `Level`, as currently implemented, relies on CSFs. We can maybe document it in #61.